lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -1177,9 +1177,9 @@ def cmd_migrate(*args)
     begin
       server = client.sys.process.open
     rescue TimeoutError => e
-      elog(e.to_s)
+      elog('Server Timeout', error:e)
     rescue RequestError => e
-      elog(e.to_s)
+      elog('Request Timeout', error:e)
```

Comment:
These don't look like the correct error messages to me

Reply:
Copypasta fat thumb